### PR TITLE
C199704: create a new request with patron comment test (Vega) (TaaS)

### PR DESCRIPTION
**Pull request by AliaksandrShuleika**
[Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4558/allure/#suites/196f4b18487f9204a68d74115e28d7e7/80d2239b3e003284/)
[Ticket](https://foliotest.testrail.io/index.php?/cases/view/199704)
[TestRail](https://foliotest.testrail.io/index.php?/cases/view/199704)

### CODE REVIEW

File: cypress/e2e/requests/new-request-with-patron-comment.cy.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import EditRequest from '../../support/fragments/requests/edit-request';
+
+describe('ui-requests: Request: Create a New Request with Patron Comment.', () => {
+  const folioInstances = InventoryInstances.generateFolioInstances();
+  const user = {};
+  const testData = {};
+
+  before(() => {
+    cy.loginAsAdmin();
```
Comment (dnovikau) [inline cypress/e2e/requests/new-request-with-patron-comment.cy.js:13]: There are permissions specified for current test:
```
Logged in as user with next permissons:

Requests: All permissions
User is on the "Requests" app
```

Temp user need to be created for the test with mentioned permissions set and deleted after the execution

**Review by dnovikau — CHANGES_REQUESTED** [review]

File: cypress/e2e/requests/new-request-with-patron-comment.cy.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        testData.servicePoint.id,
+        requestUserData.userId,
+        testData.servicePoint.id,
+      );
+      cy.login(requestUserData.username, requestUserData.password);
+    });
+  });
+
+  after('Deleting created user', () => {
+    Users.deleteViaApi(requestUserData.userId);
```
Comment (ernestyedigaryan1) [inline cypress/e2e/requests/new-request-with-patron-comment.cy.js:64]: ServicePoints, InventoryInstances, Location, PatronGroups, UserEdit should also been deleted

**Review by Scorpibear — APPROVED** [review]

File: cypress/e2e/requests/new-request-with-patron-comment.cy.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      cy.login(requestUserData.username, requestUserData.password);
+    });
+  });
+
+  after('Deleting created entities', () => {
+    UserEdit.changeServicePointPreferenceViaApi(requestUserData.userId, [testData.servicePoint.id]);
+    ServicePoints.deleteViaApi(testData.servicePoint.id);
+    cy.deleteItemViaApi(folioInstances[0].itemIds);
+    cy.deleteHoldingRecordViaApi(folioInstances[0].holdingId);
+    InventoryInstance.deleteInstanceViaApi(folioInstances[0].instanceId);
```
Comment (ernestyedigaryan1) [inline cypress/e2e/requests/new-request-with-patron-comment.cy.js:68]: I think will be better to use 
`InventoryInstances.deleteInstanceViaApi({
        instance: folioInstances[0],
        servicePoint: testData.servicePoint,
        shouldCheckIn: true,
      });`
      
It also deletes item and holding record inside of it

Comment (AliaksandrShuleika) [reply cypress/e2e/requests/new-request-with-patron-comment.cy.js]: fixed

**Review by ernestyedigaryan1 — APPROVED** [review]

**Pull request merged**